setup.py

```diff
@@ -49,16 +49,34 @@
         'Sphinx>=1.5.6,<1.6',
         'invenio-accounts>=1.0.0b1',
     ],
+    # Elasticsearch version
+    'elasticsearch2': [
+        'elasticsearch>=2.0.0,<3.0.0',
+        'elasticsearch-dsl>=2.0.0,<3.0.0',
+    ],
+    'elasticsearch5': [
+        'elasticsearch>=5.0.0,<6.0.0',
+        'elasticsearch-dsl>=5.0.0,<6.0.0',
+    ],
+    # NOTE: Latest version of elasticsearch-dsl is not compatible with ES6 yet.
+    # 'elasticsearch6': [
+    #     'elasticsearch>=6.0.0,<7.0.0',
+    #     'elasticsearch-dsl>=5.4.0.dev0,<7.0.0',
```

Comment:
``elasticsearch-dsl>=5.4.0.dev0,<7.0.0`` I guess this should be ``elasticsearch-dsl>=6.0.0,<7.0.0`` once the new version 6 is out of dsl?

Reply:
In case we decide introduce ES6 support testing and use the `master` branch of `elasticsearch-dsl` (by adding it to `requirements-devel.txt`), the current version that supports ES6 is marked as `5.4.0.dev0`... There is an ongoing discussion on https://github.com/elastic/elasticsearch-dsl-py/issues/771 about ES6 support